### PR TITLE
First version of my first blog post

### PR DESCRIPTION
**Pull request by codalicious**

### CODE REVIEW

File: source/_posts/2015-06-01-creating-and-publishing-a-ruby-gem.markdown [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+
+	lib = File.expand_path('../lib', __FILE__)
+	$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+	require 'slug_converter/version'
+
+	Gem::Specification.new do |spec|
+	  spec.name         = "slug_converter"
+	  spec.version      = SlugConverter::VERSION
+	  spec.authors      = ["Your Name"] //these ware populated by bundler 
+	  spec.email        = ["youremail@example.com"] //these ware populated by bundler
```
Comment (basti) [inline source/_posts/2015-06-01-creating-and-publishing-a-ruby-gem.markdown]: @codalicious spelling: ware => were (both lines)

File: source/_posts/2015-06-01-creating-and-publishing-a-ruby-gem.markdown [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+	rake release  # Create tag v0.0.1 and build and push slug_converter-0.1.0.gem to Rubygems
+
+
+## Writing tests
+
+If you are following the principles of Test Driven Development you will probably like to start by writing tests for you gem, for that purpose I would suggest using RSpec. 
+To do that you will need to add rspec as a development dependency to you gemspec file:
+	spec.add_development_dependency 'rspec'
+
+It was mentioned in the beginning that, when running bundle gem for the first time bundler will asks if you would like to generate test files for your gem and to choose if you want to use rspec or minitest. If you answer with yes, and choose rspec, bundler will generate a spec directory with two files:
```
Comment (basti) [inline source/_posts/2015-06-01-creating-and-publishing-a-ruby-gem.markdown]: @codalicious why so passive? :smiley: Maybe use something like: `as mentioned earlier` or `as I mentioned in the beginning.

File: source/_posts/2015-06-01-creating-and-publishing-a-ruby-gem.markdown [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+	require 'rspec/core/rake_task'
+	RSpec::Core::RakeTask.new(:spec)
+
+and then we will import this file in our Rakefile that bundler provided automatically:
+	Dir.glob('tasks/**/*.rake').each(&method(:import))
+    
+Now run:
+	bundle exec rake spec 
+
+And watch your tests fail. :)
```
Comment (basti) [inline source/_posts/2015-06-01-creating-and-publishing-a-ruby-gem.markdown:256]: @codalicious :smiley: :sunglasses:

**Comment by basti** [comment]
@codalicious Very good! :+1:

There will probably be couple formatting issues, but we can check that on local server before publishing.

**Comment by codalicious** [comment]
I made the suggested corrections and modifications and added text describing how gem can be c
configured.

**Comment by codalicious** [comment]
I forgot to ask for the instructions for publishing this blog post, or will you do it @basti ?

**Comment by basti** [comment]
We’ll do it together. I’d like to see if I remember what we had to do on
Jovica’s computer in order to get it working. :)

**Pull request merged**